dev-docker-compose.yml

```diff
@@ -1,22 +1,51 @@
 version: '3'
-
 services:
   pwa:
     build:
       context: ./pwa
     volumes:
-      - ./pwa/:/app
+      # These dirs can be changed outside of the conatiner, and reflected within.
+      # All other files require a rebuild:
+      #     docker-compose -f dev-docker-compose.yml up --build
+      - ./pwa/src:/data/src
```

Comment:
What is the motivation for only allowing certain changes to be reflected within the container? Are there directories that shouldn't be reflected when files are changed outside?

Reply:
Yep, thinks like build artifacts / node_modules